apps/heft/src/plugins/WebpackWarningPlugin.ts

```diff
@@ -53,9 +53,9 @@ export class WebpackWarningPlugin implements IHeftPlugin {
         new Error(
           'Your project appears to have a Webpack configuration generated by a plugin, ' +
             'but the Heft plugin for Webpack is not enabled. To fix this, you can add ' +
-            '"@rushstack/heft-webpack4-plugin" to your package.json devDependencies and use ' +
-            'config/heft.json to load it. For details, see this documentation: ' +
-            'https://rushstack.io/pages/heft_tasks/webpack/'
+            '"@rushstack/heft-webpack4-plugin" or "@rushstack/heft-webpack5-plugin" to ' +
+            'your package.json devDependencies and use config/heft.json to load it. ' +
+            'For details, see this documentation: https://rushstack.io/pages/heft_tasks/webpack/'
```

Comment:
Just curious -- what would happen if someone accidentally loaded BOTH plugins?

Reply:
You'll get a `A plugin with name "WebpackPlugin" has already been applied` error. We can eventually clean that error up, but I wouldn't consider that a high priority.